docs/changelog.rst

```diff
@@ -4,6 +4,12 @@ Changelog
 `Unreleased`_ - TBD
 -------------------
 
+**Fixed**
+- Worker threads may not be immediately stopped on SIGINT. `#1066`_
```

Comment:
Please, put this line under the existing `Fixed` subsection below

Reply:
It also needs `.. _#1066: https://github.com/schemathesis/schemathesis/issues/1066` line below in this file (around the line 2140)

Reply:
Sorry, I messed it up... Pushed an update.

Reply:
That's perfectly fine :) thanks